src/cider/nrepl/middleware/util/namespace.clj

```diff
@@ -52,11 +52,22 @@
      ;; rewritten by dolly
      (.startsWith ns-name "eastwood.copieddeps"))))
 
+(defn internal-namespace?
```

Comment:
This is just a regex matcher on a list now. But I think this name makes it easier to read functions where this is used.